vello_tests/tests/known_issues.rs

```diff
@@ -113,6 +114,7 @@ const DATA_IMAGE_PNG: &[u8] = include_bytes!("../snapshots/smoke/data_image_roun
 /// Test for <https://github.com/linebender/vello/issues/972>
 #[test]
 #[ignore = "CI runs these tests on a CPU, leading to them having unrealistic precision"]
+// #[cfg_attr(skip_gpu_tests, ignore)]
```

Comment:
Why is this commented out?

Reply:
Because it's an unused attribute; the `ignore` above always applies, so we'd get a warning if we uncommented this.

As it happens, I don't think that warning would block CI, but that's not particularly intentional.

Reply:
I see.

Reply:
I've added some guidance